book/version_control.md

```diff
@@ -395,140 +336,102 @@ To handle a data breach, you should:
-   * Discuss the breach with the Data Protection Officer in your department. They should be able to advise you on the steps you should take to, according to your departments data security policy.
+   * Discuss the breach with the Data Protection Officer in your department. They should be able to advise you on the steps you should take as well, according to your departments data security policy.
    * If the breach includes pre-publication statistics, you must also report the breach to the [Office for Statistics Regulation](https://osr.statisticsauthority.gov.uk/).
 
-The [Pro Git book section on rewriting history](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) details methods for editing and deleting files from your repository's commit history.
-The [BFG repo-cleaner tool](https://rtyley.github.io/bfg-repo-cleaner/) can be a simpler alternative to standard Git commands.
-
-
-## Beyond Git - GitHub
+The [Pro Git book section on rewriting history](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) details methods for editing and deleting files from your repository's commit history. The [BFG repo-cleaner tool](https://rtyley.github.io/bfg-repo-cleaner/) can be a simpler alternative to standard Git commands.
 
-A number of platforms extend the functionality of Git, to further improve collaborative working.
 
-Here we describe some of the beneficial features supported by GitHub, the world's leading software development platform.
-GitHub provides additional tools for collaborative workflows, including:
-* Access management for viewing and contributing
-* Issues
-* Project boards
-* Forking
-* Pull requests
-* Continuous Integration
+## GitHub
 
-Many of these project management tools are also discussed on the [GitHub features page](https://github.com/features/project-management/).
+A number of version control platforms extend the functionality of Git, to further improve collaborative working.
 
+Here we describe some of the beneficial features supported by [GitHub](https://github.com/), the world's leading software development platform. GitHub provides additional tools for better management of collaborative work. Many of these tools are also discussed in detail on the [GitHub features page](https://github.com/features/project-management/), but we will describe how they may be applied in analytical workflows here.
 
-### Access management
+### Efficient use of issues
 
-GitHub repositories have a [visibility setting](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/setting-repository-visibility) that represents whether the repo can be viewed publicly or only by owners of the project,
-Note that some GitHub features are limited or are not available for private repos on free GitHub accounts.
+Issues offer a method for requesting or recording tasks, including enhancements and bug fixes in a project. They act as a collaborative todo list, which users and developers can easily contribute to.
```

Comment:
compare them to trello and jira and say you can have your code and your project todos in one place, but this isn't jira so don't expect all the shiny project management ad ons.